arrow-flight/src/sql/client.rs

```diff
@@ -131,8 +131,10 @@ impl FlightSqlServiceClient<Channel> {
         self.get_flight_info_for_command(cmd).await
     }
 
-    /// Perform a `handshake` with the server, passing credentials and establishing a session
-    /// Returns arbitrary auth/handshake info binary blob
+    /// Perform a `handshake` with the server, passing credentials and establishing a session.
+    /// If the server returns an "authorization" header, it is automatically parsed and set as
```

Comment:
```suggestion
    ///
    /// If the server returns an "authorization" header, it is automatically parsed and set as
```

This just improves the doc formatting